ITEM_en.tsv

```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
 ITEM_20150612_005304	Green Kokateu Bone
 ITEM_20150612_005305	A Green Kakatue bons is very hard so it is mainly used to make the arrowheads.
 ITEM_20150612_005306	Colson Signal Bell
-ITEM_20150612_005307	A tool send the signal to his companion.
+ITEM_20150612_005307	A tool Colson used to send a signal to his companion.
```

Comment:
"Tools that the Hunters used to send signals to each other. Curiously, their Companions only respond to their owners' bells."